corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -143,9 +143,11 @@ public int printTable(String namespace, String tablename) {
                     builder = new StringBuilder("\nKey:\n")
                             .append(JsonFormat.printer().print(entry.getKey().getKey()))
                             .append("\nPayload:\n")
-                            .append(JsonFormat.printer().print(entry.getValue().getPayload()))
+                            .append(entry.getValue() != null && entry.getValue().getPayload() != null ?
+                                    JsonFormat.printer().print(entry.getValue().getPayload())   : "")
```

Comment:
Print null instead of empty string?

Reply:
done. thanks

Reply:
Actually no, we can't print null. null does not get parsed in the same way as an empty string.
We will need this output of the browser to be read back in, and strings need to be printed as empty because that's how protobufs stores strings